In [1]:
# imports
import pandas as pd
# import matplotlib.pyplot as plt
import numpy as np

In [11]:
# dataset 1 
data_info = pd.read_csv("./metadata.csv")
data_info = data_info.fillna(0)
data_info

sample_id  split  derivatized             features_path  \
0        S0000  train          0.0  train_features/S0000.csv   
1        S0001  train          0.0  train_features/S0001.csv   
2        S0002  train          1.0  train_features/S0002.csv   
3        S0003  train          0.0  train_features/S0003.csv   
4        S0004  train          0.0  train_features/S0004.csv   
...        ...    ...          ...                       ...   
1116     S1116    val          1.0    val_features/S1116.csv   
1117     S1117    val          0.0    val_features/S1117.csv   
1118     S1118    val          0.0    val_features/S1118.csv   
1119     S1119    val          1.0    val_features/S1119.csv   
1120     S1120    val          0.0    val_features/S1120.csv   

                     features_md5_hash  
0     52ec6d6f8372500ab4e069b5fbdae6f9  
1     348f90baed8a8189bf0d4c7b9ed9f965  
2     4686ad9bc3716966f63b6ff83d1d8324  
3     de6b53605c5887967dc3661a3a711c2b  
4     fbfd90092d10d15a5d6919327ddde2ab  
...                                ...  
1116  86a7ab7517b69448fd6eb6f518670c5b  
1117  82ad1a2c511290193697b4dc6baa131e  
1118  2125667ba387f06d20e644f2a612c8a0  
1119  d8f27d84c0bf0a888ebf7cbb4f832624  
1120  6bed4722f14301197d499fc9c0a969ba  

[1121 rows x 5 columns]

In [13]:
# for sample S0000 - first sample
data_info_train = data_info.loc[data_info.split=='train']
data_info_val = data_info.loc[data_info.split=='val']


In [16]:
data_info_val

sample_id split  derivatized           features_path  \
809      S0809   val          1.0  val_features/S0809.csv   
810      S0810   val          0.0  val_features/S0810.csv   
811      S0811   val          0.0  val_features/S0811.csv   
812      S0812   val          0.0  val_features/S0812.csv   
813      S0813   val          0.0  val_features/S0813.csv   
...        ...   ...          ...                     ...   
1116     S1116   val          1.0  val_features/S1116.csv   
1117     S1117   val          0.0  val_features/S1117.csv   
1118     S1118   val          0.0  val_features/S1118.csv   
1119     S1119   val          1.0  val_features/S1119.csv   
1120     S1120   val          0.0  val_features/S1120.csv   

                     features_md5_hash  
809   89454e8f682608645d89f7610d4ad958  
810   905059919c8da350146850a981c92b4a  
811   71260a24237d740e31fe3d5dc28d8d29  
812   63c0afd0bb4064bf485861af347dbca8  
813   d94c4e367d9a414d0a3cc602d164b74d  
...                                ...  
1116  86a7ab7517b69448fd6eb6f518670c5b  
1117  82ad1a2c511290193697b4dc6baa131e  
1118  2125667ba387f06d20e644f2a612c8a0  
1119  d8f27d84c0bf0a888ebf7cbb4f832624  
1120  6bed4722f14301197d499fc9c0a969ba  

[312 rows x 5 columns]

Train data 809 rows with labels \
Val   data 312 rows with no labels -- we have to predict this data \
total 1221 rows
